packages/generator-widget/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+- We've updated our templates to support the new version of Pluggable Widgets Tools containing new React and React Native versions.
```

Comment:
I think we should mention explicitly in both changelogs that this is necessary because of breaking changes in Mendix 9.24.

Reply:
Done